pkg/skaffold/build/local/types.go

```diff
@@ -85,12 +86,17 @@ func NewBuilder(bCtx BuilderContext, buildCfg *latestV1.LocalBuild) (*Builder, e
 	}
 
 	cluster := bCtx.GetCluster()
+	pushFlag := bCtx.PushImages()
 
 	var pushImages bool
-	if buildCfg.Push == nil {
+	switch {
+	case pushFlag.Value() != nil:
+		logrus.Debugf("push value set via skaffold build --push flag, --push=%t", *pushFlag.Value())
+		pushImages = *pushFlag.Value()
```

Comment:
tiniest nit: keep ordering of log message and `pushImages` assignment consistent with the `case` below this one

Reply:
Fixed